components/service/glean/src/main/AndroidManifest.xml

```diff
@@ -2,9 +2,22 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.components.service.glean" >
+          package="mozilla.components.service.glean" >
+
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
```

Comment:
@pocmo what's your opinion of glean requiring this permission? Is this reasonable for a library to come with such requirements?

Reply:
Resolving this, as we noticed that the legacy telemetry library uses the same permission.